SpringProject/src/main/java/com/gradproject2019/users/controller/UserController.java

```diff
@@ -24,4 +24,16 @@
         UserResponseDto newUser = userService.saveUser(userRequestDto);
         return ResponseEntity.ok(newUser);
     }
+
+    @PatchMapping(path = "/users/{id}")
+    public ResponseEntity<UserResponseDto> editUser(@RequestBody UserPatchRequestDto userPatchRequestDto, UserRequestDto userRequestDto, @PathVariable("id") Long userId) {
```

Comment:
Why do you need both UserPathRequestDto and UserRequestDto? What about the token for authorization?

Reply:
This is something outside the scope of this ticket. @MerrettM is working on this in a separate ticket

Reply:
That answers the second part of the question, what about the first one?

Reply:
Both of those are part of my ticket's scope, the first part of the question is because that's wrong and I've fixed it, the second part is that I'm currently implementing this in my PR.